client/src/components/ratingsystem/CreateRatingSystemForm.tsx

```diff
@@ -0,0 +1,261 @@
+import { 
+    Input, 
+    Radio, 
+    Stack,
+    VStack,
+    Button,
+    Heading,
+    FormControl, 
+    FormLabel,
+    FormErrorMessage,
+    FormHelperText,
+    RadioGroup,
+    Center,
+    NumberDecrementStepper,
+    NumberIncrementStepper,
+    NumberInput,
+    NumberInputField,
+    NumberInputStepper
+} from "@chakra-ui/react";
+
+import * as React from 'react';
+import{ Formik, Form, Field, useFormik } from "formik";
+import * as Yup from 'yup';
+
+// name: String!
+// type: String!
+// lowerBound: Int!
+// upperBound: Int!
+// labels: [String!]!
+// subrating: String!
+// subratings: [Subrating!]!
+
+function contElements(){
```

Comment:
These seem to either show or hide the elements. We could clean this up later with state variables instead of having separate functions to show/hide elements

Reply:
Alright, I created a new issue that addresses this